src/panels/lovelace/entity-rows/hui-button-entity-row.ts

```diff
@@ -0,0 +1,82 @@
+import "@material/mwc-button/mwc-button";
```

Comment:
What is the difference in the special button row and this? Could we combine them or is there a reason for two

Reply:
One is based on an entity, and the other is just config